mock-etcd/src/mock_etcd.rs

```diff
@@ -263,6 +263,38 @@ impl MockEtcdInner {
         }
     }
 
+    /// Send watch response if client falls behind
+    async fn catch_up_revision(
```

Comment:
I think it will send back response, containing all the related changes between watch revision id and current revision id.

Reply:
Correct. But mock-etcd doesn't need implement that which needs log, so we just send the latest value.